### PR TITLE
Chore(2.9.0): Update helm docs for version 2.9.0

### PR DESCRIPTION
**Pull request by uditgaurav**
Signed-off-by: uditgaurav <udit@chaosnative.com>

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [ ] DCO signed
- [ ] Chart Version bumped
- [ ] Variables are documented in the README.md

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,201 +1,674 @@
-                                 Apache License
-                           Version 2.0, January 2004
```
Comment (Jonsy13) [inline LICENSE:2]: Is this change required?

File: README.md [hunk 2]
```diff
@@ -1,111 +1,416 @@
-# Litmus helm
+helm-docs
```
Comment (Jonsy13) [inline README.md:1]: Is this change required?

**Review by Jonsy13 — CHANGES_REQUESTED** [review]

**Review by Jonsy13 — APPROVED** [review]

**Review by Jasstkn — APPROVED** [review]
